org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/customservice/synapse/parser/ConfigDetails.java

```diff
@@ -14,11 +14,37 @@
 package org.eclipse.lemminx.customservice.synapse.parser;
 
 import org.eclipse.lsp4j.Range;
+import org.eclipse.lsp4j.jsonrpc.messages.Either;
+
+import java.util.List;
 
 public class ConfigDetails {
 
     public String key;
+    public String type;
     public String value;
-    public Range range;
+    public Either<Range, List<Range>> range;
```

Comment:
shall we declare these as private variables?

Reply:
Fixed in eee1f630f307a175d76282ac19bd2550adf1c996